anomalib/models/patchcore/model.py

```diff
@@ -46,18 +46,20 @@ def __init__(
         self.input_size = input_size
         self.sigma = sigma
 
-    def compute_anomaly_map(self, patch_scores: torch.Tensor) -> torch.Tensor:
+    def compute_anomaly_map(self, patch_scores: torch.Tensor, feature_map_shape: torch.Tensor) -> torch.Tensor:
```

Comment:
@alexriedel1, is `feature_map_shape` of type `torch.Tensor` or `torch.Size`?

Reply:
Right its a `torch.Size`

Reply:
Thanks for quickly addressing it